tools/topology/sof/tokens.m4

```diff
@@ -88,3 +88,10 @@ SectionVendorTokens."sof_tone_tokens" {
 SectionVendorTokens."sof_effect_tokens" {
 	SOF_TKN_EFFECT_TYPE			"900"
 }
+
+SectionVendorTokens."sof_ch_sel_tokens" {
```

Comment:
@keyonjie we dont need this anymore. lets drop it.

Reply:
let me drop it with FW change and bytes control ready for channel selector and kpbm ready later. @juimonen